pkg/helpers/units.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+package helpers
+
+import (
+	"fmt"
+	"math"
+)
+
+func Bytes(s uint64) string {
```

Comment:
Where is this func being called?

Reply:
I removed the consumer code and forgot to delete it